aquarius/events/purgatory.py

```diff
@@ -34,7 +34,16 @@ def retrieve_new_list(self, env_var):
             return {
                 (a["did"], a["reason"]) for a in response.json() if a and "did" in a
             }
-
+            if env_var == "ASSET_PURGATORY_URL":
```

Comment:
env_var will never equal "ASSET_PURGATORY_URL", it's just the url that is fetched to get the purgatory list

Reply:
That's not correct. The env_var is set in line 99 to _ASSET_PURGATORY_URL_ and in line 103 to _ACCOUNT_PURGATORY_URL_.  The URL is in the Environment variable called env_var. Like you used in line 28 _response = requests.get(os.getenv(env_var), timeout=5)_. The env_var will depend on the lines 99 and 103. It just switched what it needs to return based on the environment variable name, since the Asset and Account Purgatory URLs have different layouts.

Reply:
I removed the return line before if env_var statements. Added it again by accident doing black formating. 

The env_var is only the Environment variable name not the URL itself. The URL is in the Environment see line 28 (os.getenv(env_var)).

This fixes the error that the _new_account_list_ was always empty. Since it doesn't has the _did_:_reason_ instead _account_:_reason_.